packages/datacommons-api/pyproject.toml

```diff
@@ -1,55 +1,31 @@
-[build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
-
 [project]
 name = "datacommons-api"
 dynamic = ["version"]
 description = 'Data Commons API'
+license = "Apache-2.0"
 readme = "README.md"
-requires-python = ">=3.11"
-license = "MIT"
+requires-python = ">=3.11,<3.13"
```

Comment:
I think it's generally not recommended to set an upper cap on the Python version. Consider removing?